### ISSUE TITLE
What is a recipe ?

### ISSUE THREAD

**Issue by Aaishpra** [conda-forge/conda-forge.github.io#1389]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

### Issue:
Throughout the conda-forge documentation, there is a mention of the word recipes, How to add a recipe, how to update a recipe,
But no mention about what is a recipe for packages, in my view there should be some sort of small introduction about what is a recipe and it can be added at the homepage.

**Comment by Aaishpra** [comment]
@viniciusdc if this issue seems legit and needed I can work here or you can close the issue thanks

**Comment by beckermr** [comment]
A recipe is the set of files needed to build the conda package. This includes at minimum the meta.yaml, but might also include license files, patches, build scripts, test scripts etc. If you look in any feedstock, the recipe is all of the stuff in the recipe directory.

**Comment by Aaishpra** [comment]
Yes, but can that be included on homepage so someone visiting the website for the first time can directly know what it is.

**Comment by beckermr** [comment]
Sure. I'd make a glossary entry and add some links

**Comment by viniciusdc** [comment]
> Yes, but can that be included on homepage so someone visiting the website for the first time can directly know what it is.

Even though we don't have a straighforward directly definition of what a recipe is, it's conteints and uses are well written in the docs. There is still a full exemple of it (see more [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml)

**Comment by Aaishpra** [comment]
Ya, I saw it and it perfectly explained me how a recipe can be created or what it's contents are, but due to the excessive use of the word recipe, i feel there should be some proper definition of this word and this definition should be present at the homepage.

**Comment by viniciusdc** [comment]
> Ya, I saw it and it perfectly explained me how a recipe can be created or what it's contents are, but due to the excessive use of the word recipe, i feel there should be some proper definition of this word and this definition should be present at the homepage.

sure, I totally understand. I think the best answer to this was given by Matt above, also we could adapt the definition from [conda-recipes](https://docs.conda.io/projects/conda-build/en/latest/concepts/recipe.html) and use this chance to explain what the final structure of the feedstock will be... @beckermr any thoughts on this?

**Comment by beckermr** [comment]
Yup. It is anything in the recipe folder in a feedstock. It always has a meta.yaml. It can have a bunch of other things.

**Comment by Aaishpra** [comment]
@beckermr @viniciusdc can I work on this and add this definition you and @viniciusdc provided under a new heading named "What is a Recipe" on the homepage. Although I am still unsure what could be the best place to add it, but [this](https://conda-forge.org/index.html#about) can be a somewhat better place.

**Comment by beckermr** [comment]
Please make a glossary entry and link the term to that.
